src/view/com/notifications/FeedItem.tsx

```diff
@@ -56,15 +56,15 @@ interface Author {
   moderation: ProfileModeration
 }
 
-export function FeedItem({
+let FeedItem = ({
```

Comment:
hi, i'm just stranger
but can you explain this approach, please? interesting
why you need to store component in let variable and apply memo later, instead of just wrapping component in memo?

Reply:
wow, i doesn't know that every stranger can comments on pr's like that
if i'm not allowed to, please sorry

Reply:
In the react docs, [memo](https://react.dev/reference/react/memo) accepts a component to memoize, which is in this case a funcitonal component. You can either store the component's function in a variable then pass it to memo (what is done here), or just inline it in the `memo()` call (what is done in [the docs](https://react.dev/reference/react/memo#memo).

Reply:
It doesn't really matter, I just want to avoid an extra indentation level in the code.